src/FileProviders/Manifest.MSBuildTask/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
Another example of a project that has public API, but the analyzer isn't running on.

Reply:
Project file contains `<IsImplementationProject>false</IsImplementationProject>` so it's excluded from the analysis